README.md

```diff
@@ -14,15 +14,18 @@ To use this module, follow these instructions to [add a module from the Viam Reg
 
 Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
 Click on the **Services** subtab and click **Create service**.
-Select the `mlmodel` type, then select the `onnx-cpu:timesyncsensor` model.
+Select the `mlmodel` type, then select the `mlmodel:onnx-cpu` model.
 Click **Add module**, then enter a name for your ML model and click **Create**.
 
-On the new component panel, copy and paste the following attribute template into your ML model’s **Attributes** box:
+On the new component panel, deploy a model on to your machine by selecting an onnx model from the **Model** dropdown or by specifying a path to an existing model on your machine.
+
+If you prefer to configure your service using JSON, use the following attributes **attributes**:
 
 ```json
 {
   "model_path": "/path/to/onnx_file/detector_googlenet.onnx",
-  "label_path": "/path/to/labels.txt"
+  "label_path": "/path/to/labels.txt",
+  "num_threads": 1
```

Comment:
num_threads is not actually a config attribute for this model  - onnx models can only run on 1 thread

Reply:
Should it be removed from the app UI then?

Reply:
Because it does show up...